libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -922,6 +1025,69 @@ string YulUtilFunctions::updateStorageValueFunction(Type const& _type, boost::op
 	});
 }
 
+string YulUtilFunctions::updateMemoryValueFunction(Type const& _type)
+{
+	if (auto ref = dynamic_cast<ReferenceType const*>(&_type))
+	{
+		solUnimplementedAssert(
```

Comment:
```suggestion
		solAssert(
```

(this function is called `updateMemoryValueFunction` - the reason string should also be updated.